packages/headless/src/composables/tooltip/createTooltip.ts

```diff
@@ -1,19 +1,116 @@
+import { computed, ref, unref, watchEffect, type MaybeRef } from "vue";
 import { createId } from "../..";
 import { createBuilder } from "../../utils/builder";
 
-export const createTooltip = createBuilder(() => {
+export type CreateTooltipOptions = {
+  trigger: MaybeRef<TooltipTrigger | boolean>;
+  /**
+   * Number of milliseconds to use as debounce when showing/hiding the tooltip
+   * with openMode "hover".
+   *
```

Comment:
A delay of `0` can be annoying, so I would prefer something like `1000`

Reply:
I set a default value of 200 which can be overridden if needed